lib/src/sql/duration.rs

```diff
@@ -27,7 +27,7 @@ static NANOSECONDS_PER_MICROSECOND: u32 = 1000;
 
 pub(crate) const TOKEN: &str = "$surrealdb::private::sql::Duration";
 
-#[derive(Clone, Debug, Default, Eq, PartialEq, PartialOrd, Serialize, Deserialize, Hash)]
+#[derive(Clone, Copy, Debug, Default, Eq, PartialEq, PartialOrd, Serialize, Deserialize, Hash)]
 #[serde(rename = "$surrealdb::private::sql::Duration")]
 #[revisioned(revision = 1)]
 pub struct Duration(pub time::Duration);
```

Comment:
I am puzzled here. time::Duration is a pretty compact structure. But I often hesitate in giving `Copy`, and force developers to use `clone` instead, so the developer knows that he is running a possibly costly operation. If I am not wrong time::Duration does not implement 'Copy'

Reply:
> If I am not wrong time::Duration does not implement 'Copy'

[It does](https://docs.rs/time/0.3.29/time/struct.Duration.html#impl-Copy-for-Duration) @emmanuel-keller, otherwise it wouldn't be possible to derive `Copy` here.